### DISCUSSION TITLE
Member Browser

### DISCUSSION THREAD

**Discussion by Flieh** [codefori/vscode-ibmi#429] [Q&A] [answered]
Not sure what I'm doing wrong but when I update the extension beyond v0.7.xx the Member Browser disappears. I couldn't find anything on the issues page which makes me think my settings are the problem. Any ideas? Thanks

**Comment by Flieh** [comment, marked as answer]
Ok sorry, I'd been rattling around for the answer and then I did find it in issues. I need to create a custom filter in the object browser.

**Reply by worksofliam** [reply]
Yup, you got it. The member browser was removed and the source browser does everything now.
